### PR TITLE
Capturing load time stats

### LINKED ISSUE #15141: Capture data about file load times

### PR DESCRIPTION
**Pull request by jpuri**
Fixes: #15141

Capture file load time stats for MV3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "start:test": "SEGMENT_HOST='https://api.segment.io' SEGMENT_WRITE_KEY='FAKE' yarn build testDev",
     "start:test:mv3": "ENABLE_MV3=true SEGMENT_HOST='https://api.segment.io' SEGMENT_WRITE_KEY='FAKE' yarn build testDev",
     "benchmark:chrome": "SELENIUM_BROWSER=chrome node test/e2e/benchmark.js",
+    "mv3:stats:chrome": "SELENIUM_BROWSER=chrome node test/e2e/mv3/stats.js",
```
Comment (segun) [inline package.json:21]:
```suggestion
    "mv3:stats:chrome": "SELENIUM_BROWSER=chrome ENABLE_MV3=true node test/e2e/mv3/stats.js",
```

File: test/e2e/webdriver/chrome.js [hunk 2]
```diff
@@ -18,7 +18,13 @@ class ChromeDriver {
     if (responsive) {
       args.push('--auto-open-devtools-for-tabs');
     }
-    args.push('--log-level=3');
+    // if (process.env.ENABLE_MV3 !== 'false') {
```
Comment (segun) [inline test/e2e/webdriver/chrome.js:21]:
```suggestion
    if (process.env.ENABLE_MV3) {
```

File: test/e2e/webdriver/chrome.js [hunk 3]
```diff
@@ -18,7 +18,13 @@ class ChromeDriver {
...
+    args.push('--log-level=0');
+    args.push('--enable-logging');
+    args.push(`--user-data-dir=${process.cwd()}/test-artifacts/chrome`);
+    // } else {
```
Comment (segun) [inline test/e2e/webdriver/chrome.js:25]:
```suggestion
     } else {
         args.push('--log-level=3');
     }
```

**Review by segun — CHANGES_REQUESTED** [review]

**Comment by Milk1235** [comment]
> Fixes: #15141
> 
> Capture file load time stats for MV3

**Review by segun — DISMISSED** [review]
LGTM

**Review by brad-decker — DISMISSED** [review]

**Review by seaona — DISMISSED** [review]
@jpuri I think we should update `package.json` to point into the new location and update the script name for mv3 test:

Now:
`"mv3:stats:chrome": "SELENIUM_BROWSER=chrome ENABLE_MV3=true node test/e2e/lavamoat-stats.js",`

Change needed:
`"mv3:stats:chrome": "SELENIUM_BROWSER=chrome ENABLE_MV3=true node test/e2e/mv3/stats.js",`

**Review by brad-decker — APPROVED** [review]

**Review by segun — APPROVED** [review]

**Pull request merged**